keystone_scim/contrib/scim/core.py

```diff
@@ -36,8 +36,8 @@
     ]
 }
 
-extension.register_admin_extension(EXTENSION_DATA['alias'], EXTENSION_DATA)
-extension.register_public_extension(EXTENSION_DATA['alias'], EXTENSION_DATA)
+#extension.register_admin_extension(EXTENSION_DATA['alias'], EXTENSION_DATA)
+#extension.register_public_extension(EXTENSION_DATA['alias'], EXTENSION_DATA)
```

Comment:
Leftovers?

Reply:
fixed in https://github.com/telefonicaid/fiware-keystone-scim/pull/63/commits/4b29f070f4eec20f8ee2bdad2810b01d8cd92198